pymess/locale/cs/LC_MESSAGES/django.po

```diff
@@ -582,3 +581,19 @@ msgstr "zakázané objekty SMS zpráv"
 #: utils/html.py:23
 msgid "HTML body contains one of banned tag: {}"
 msgstr "Tělo HTML obsahuje jeden ze zakázaných tagů: {}"
+
+#: models/emails.py:151 models/emails.py:286
+msgid "send as"
+msgstr "odeslat jako"
+
+#: models/emails.py:297
+msgid "The \"Send as\" filename must be of the same mimetype as \"File\"."
```

Comment:
tohle už tam nemáš ne? si jsi zapomněl přegenerovat

Reply:
JJ, našel jsem to a dostranil.